tools/travis/setup-mypy

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+set -e
+set -x
+pip install -r py3_test_reqs.txt
```

Comment:
add `--no-deps` so we don't install a bunch of other stuff that's not specificed explicitly if we add more things in the future.